src/main/java/org/umssdiplo/automationv01/core/utils/PropertyAccessor.java

```diff
@@ -53,15 +53,11 @@ public String getBrowser() {
         return getDataProperty(BROWSER);
     }
 
-    public String getBaseUrl() {
-        return getDataProperty(BASE_URL);
-    }
+    public String getBaseUrl() { return getDataProperty(BASE_URL); }
 
     public String getUser() {
         return getDataProperty(USER);
     }
 
-    public String getPassword() {
-        return getDataProperty(PASSWORD);
-    }
+    public String getPassword() { return getDataProperty(PASSWORD); }
```

Comment:
you can format your code before sent to code review
IDE helps us on that use the following command lines:
- ctrl + alt + L
- ctrl + alt + o

This method was defined in just one line.